src/faldbt/lib.py

```diff
@@ -67,20 +69,28 @@ def initialize_dbt_flags(profiles_dir: str):
 
 # NOTE: Once we get an adapter, we must call `connection_for` or `connection_named` to use it
 def _get_adapter(
-    project_dir: str, profiles_dir: str, profile_target: str
+    project_dir: str,
+    profiles_dir: str,
+    profile_target: str,
+    config: RuntimeConfig = None,
```

Comment:
why do we need this?

Reply:
A GREAT QUESTION! Short answer: I don't know 🚩🚩🚩

I don't need it when running tests locally but for some reason, running things in gh actions is different. 

Check this log: https://github.com/fal-ai/fal/runs/6679158525?check_suite_focus=true

For some reason it cannot find the profile dir anymore and this happens only after `compile_sql` is called and I only see it in an action worker.